cassandra/src/test/scala/geotrellis/spark/CassandraTestEnvironment.scala

```diff
@@ -39,6 +42,12 @@ trait CassandraTestEnvironment extends TestEnvironment { self: Suite =>
         println("\u001b[0;33mA script for setting up the Cassandra environment necessary to run these tests can be found at scripts/cassandraTestDB.sh - requires a working docker setup\u001b[m")
         cancel
     }
+    startTime = System.currentTimeMillis()
```

Comment:
Yes, I agree, probably benchmarks can be placed here: https://github.com/locationtech/geotrellis/tree/master/bench

Reply:
Works for me - I'll see what I can do about adding some Read, Write, and Mixed benchmarks for Cassandra under both schema strategies.

Reply:
Indeed; wondering how much differently a PK change would behave in tests on reads; since we're not performing range queries: https://github.com/locationtech/geotrellis/blob/master/cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraRDDReader.scala#L80

Reply:
Agreed entirely - I don't want to introduce more overhead or complexity if it's not to the benefit of performance.

Reply:
My thought is that we may benefit from Cassandra's row cache even across multiple queries if those queries land in the same partition.  I must admit though that I can't confirm whether or not that's actually a reasonable expectation.